client/scanner.js

```diff
@@ -120,12 +121,23 @@ class Scanner {
   }
 
   async watchBlockchain() {
-    const latestBlock = await this.getBlock('latest')
-    const startBlock = latestBlock.number - (this.config.scanSpread * 2)
+    this.web3.currentProvider.sendAsync({
+      jsonrpc: '2.0', id: 1, method: 'eth_getFilterLogs', params: []
+    }, async (e) => {
 
-    this.log.debug(`Watching for new Requests from | block: ${startBlock} `)
+      if (e !== null) {
+        this.log.info(`Watching DISABLED`)
+        return;
+      }
+
+      const latestBlock = await this.getBlock('latest')
+      const startBlock = latestBlock.number - (this.config.scanSpread * 2)
```

Comment:
Why are we multiplying `scanSpread` by two here? I believe that `latestBlock.number - this.config.scanSpread` is correct. The `scanSpread` was only getting multiplied by two because before it was counted twice for `scanSpread` amount after the block _and_ before the block. Now that we're just watching for all new requests we can remove the `* 2`